spec/forms/hyrax/forms/admin/collection_type_form_spec.rb

```diff
@@ -1,11 +1,17 @@
 RSpec.describe Hyrax::Forms::Admin::CollectionTypeForm do
+  let(:collection_type) { build(:collection_type) }
   let(:form) { described_class.new }
 
-  describe ".title" do
-    subject { form.title }
+  subject { form }
```

Comment:
@jeremyf `validate_presence_of` isn't available.  From what I read it sounds like it's only available on specs of type: :model.  Doesn't look like Hyrax is using it in any existing specs.